recipes/ckanweb-deploy-exts.rb

```diff
@@ -39,6 +39,7 @@
 virtualenv_dir = "/usr/lib/ckan/default"
 python = "#{virtualenv_dir}/bin/python"
 pip = "#{virtualenv_dir}/bin/pip --cache-dir=/tmp/"
+ckan_cli = "#{virtualenv_dir}/bin/ckan_cli"
```

Comment:
is this called after ckan-deploy.rb?

Reply:
Actually it's called *by* `ckan-deploy.rb`, near the end of the recipe. (Previously line 181, now line 188.)